recipe/meta.yaml

```diff
@@ -8,70 +8,70 @@ package:
 source:
   - folder: go
     url: https://dl.google.com/{{ name }}/go{{ version }}.src.tar.gz
-    sha256: 7bfa7e5908c7cc9e75da5ddf3066d7cbcf3fd9fa51945851325eebc17f50ba80
+    sha256: 27871baa490f3401414ad793fba49086f6c855b1c584385ed7771e1204c7e179
     patches:
       # Please see patches/README.md for more details
```

Comment:
Can this readme be updated too?

Reply:
ok, I missed it. I will provide it from CF

Reply:
hmm, actually this information is out of date on conda-forge too.  I will see, if I can rework it a bit, so it actually fits more to the new version.

Reply:
I add a summary with best practice of porting, and describing the points need to be addressed by the patches.
I keep the old information as is. So we document the intial creation of some patches